pyproject.toml

```
@@ -55,7 +55,6 @@ ignore = [
   "RUF005",
   "RUF012",
   "SIM108",
```

```suggestion
  "SIM108",
  "SIM115",
```
Restoring the rule for now so we can get a merge